tests/__init__.py

```diff
@@ -1,6 +1,10 @@
 # -*- coding: utf-8 -*-
 
-import distutils
+try:
+    import distutils.dir_util
+    import distutils.sysconfig
+except ImportError:
+    import distutils  # Python < 3.10
```

Comment:
```suggestion
import setuptools

import distutils.dir_util
import distutils.sysconfig
```

Reply:
I can't test if that works with older python versions.

Reply:
It should work with older Pythons, but likely not older setuptools. I think be need both the setuptools-first import, and the try/except.

Reply:
Yeah, that sounds more plausible but why do we need setuptools? CI and my local tests pass without it.

Reply:
Because you are always supposed to import setuptools before distutils. It patches distutils when you import it. Most builders (like pip) do this for you already, but it's good to always do it first in case they don't.

Reply:
amended and pushed